src/css/components/nav-secondary.css

```diff
@@ -37,13 +37,14 @@
     padding: 0;
     display: flex;
     align-items: center;
+    flex-direction: row-reverse;
```

Comment:
My initial thought is I'm not sure why the links are reversed between desktop and mobile. But I'll followup w the designers and fix in another PR.